store/templates/store/loaned_books.html

```diff
@@ -19,25 +19,88 @@ <h3>Loaned Books list</h3>
         <th scope="col">Genre</th>
         <th scope="col">Borrow Date</th>
         <th scope="col">Return Buttons</th>
+        <th scope="col">Rate BOOk</th>
+        <th scope="col">submit rating</th> 
         </tr>
     </thead>
     <tbody>
-        {% for copy in books %}
+        {% for copy in books %}‹
         <tr>
         <th scope="row">{{ forloop.counter }}</th>
         <td><a href="/book/{{copy.book.id}}/">{{ copy.book.title }}</a></td>
         <td>{{ copy.book.author }}</td>
         <td>{{ copy.book.genre }}</td>
         <td>{{ copy.borrow_date }}</td>
         <td><button class="btn btn-primary" onclick="returnBook({{ copy.id }})">Return {{ copy.book.title }}</button></td>
+        <td><input text="integer" class='form-text btn-primary' id="rat"></td>
+        <td><button class="btn btn-primary" onclick="rateBook({{ copy.id }})" text="number">Rate!!</button></td>
         </tr>
         {% endfor %}
+        
     </tbody>
+    <h4>rating should be between 1-10 and in integer OR else accha ni hoga</h4>
 </table>
+
+
+
 <script>
 // Fill in this function by yourself. It should make a post call to the returnBookView and display an appropriate message
-function returnBook(bid){
-    ;
-}
+    function returnBook(bid){
+        console.log("works");
+        $.ajax({
+            url: "{% url 'return-book' %}",
+            method: "POST",
+            data: {
+                bid:bid
+            },
+            success: function(data, status, xhr){
+                alert(data['message']);
+                location.reload()
+
+
+            },
+            error: function(xhr, status, err){
+                alert("integer smjhte ho wahi dalna he");
+            }
+
+        })
+
+        ;
+    }
+
+    function rateBook(bid){
+        console.log("works");
+        rating=$("#rat").val();
+        
+        rating_int=parseInt(rating,10)
+        console.log(rating,rating_int)
+        
+        if(rating==""||rating<1||rating>10){
+            alert("Rating must be between 1-10 in integer")
+        }
```

Comment:
Here, the value of `rating` is set using `#rat`, and for every book copy loaned, the id of the text-field is `rat`. Therefore, this will produce an error on rating the book. The rating stored in the 1st textbox with id `#rat` will be taken.